packages/components/src/Section/src/SectionContext.ts

```diff
@@ -0,0 +1,8 @@
+import { createContext } from "react";
+import type { ContextValue } from "react-aria-components";
+
+import type { SectionProps } from "./Section.tsx";
+
+export const SectionContext = createContext<ContextValue<SectionProps<object>, HTMLElement>>({});
```

Comment:
find an alternative to hardcoding <object> here also

Reply:
I think i actually saw this in RAC somewhere which is why I put it. not sure what else to do... but I'll check again.

Reply:
First time we have this, so i'm curious. Do you best, if it stays like this because RAC does it, it's fine